### ISSUE TITLE
Reference error in new version (1.2.0)

### ISSUE THREAD

**Issue by shtrih** [P5-wrapper/react#29]
I updated the library, but now I get an error `ReferenceError: p5 is not defined`:
![chrome_2019-06-02_06-56-54](https://user-images.githubusercontent.com/544063/58756165-f26aea80-8503-11e9-9398-0fb505e47a90.png)

**Comment by and-who** [comment]
Hi, I removed p5 as a Dependency of the p5-wrapper so everybody can choose which p5 Version he is using.

So you need to add p5 as an Dependency by yourself.

Try:
`npm install p5 --save`

**Comment by shtrih** [comment]
Yes it's already in my dependencies and looks like the exception thrown from `p5.dom`

**Comment by and-who** [comment]
I just removed the addons. (I was not sure if I should add them anyway)
Can you try Version 1.2.1?

**Comment by shtrih** [comment]
I'm sorry I found the problem. 
The problem is in `p5.dom.js`. Someone added 2 functions but in wrong place. This two functions should have been added before `});` (line 3118).
https://github.com/processing/p5.js/commit/9b5094a9575108fd4c924a01929590034832c4b0#diff-84dfcaf5d56a24f4ee13a32c644817b0R3118
Those changes was released in 21 Jan in `0.7.3`. Then this problem was fixed 23 Jan (https://github.com/processing/p5.js/commit/8c75c7346fdbeca8a22a5d47a1f887394687452d#diff-84dfcaf5d56a24f4ee13a32c644817b0). But this fix were in the next release `0.8.0`, which took place after 3 months on _9 Apr_!

1. I started my project in march so I used latest p5.js `0.7.3`. 
2. I did not use the library `p5.dom.js` so the error did not occur. 
3. Then I added `react-p5-wrapper` it uses own `p5` package `0.5.5` so the error did not occur when I started using of `p5.dom`.
4. When I updated `react-p5-wrapper` to `1.2.0` it began to use `p5` from my project which is `0.7.3`. I have created this Issue.
5. I have updated `p5` to `0.8.0` and error is gone.

Now I update react-p5-wrapper to `1.2.1`. So to use dom library I import corresponding file in my sketch.js: `import "p5/lib/addons/p5.dom";`.
🤯

Thank you.
